src/locales/en.ts

```diff
@@ -158,6 +158,12 @@ export default {
     selection: {
       title: 'Choose\nyour Hotspot.',
       subtitle: 'What kind of Hotspot do you\nwish to add?',
+      makerAppAlert: {
+        title: 'Download Now',
+        body:
+          '{{maker}} hotspots can now be onboarded using the {{makerAppName}}',
```

Comment:
Instead of `can now` maybe it should be `should now`. We won't allow them to use the helium app anymore right? @cokes518 maybe need your review on these strings.